pkg/manager/manager_test.go

```diff
@@ -50,93 +57,161 @@ const (
 	ValueLeaf2D314 = 3.14
 )
 
-// TestMain should only contain static data.
-// It is run once for all tests - each test is then run on its own thread, so if
-// anything is shared the order of it's modification is not deterministic
-// Also there can only be one TestMain per package
-func TestMain(m *testing.M) {
-	log.SetOutput(os.Stdout)
-	os.Exit(m.Run())
-}
-
-func setUp(t *testing.T) (*Manager, map[string]*change.Change, map[store.ConfigName]store.Configuration,
-	*mockstore.MockDeviceStore, *mockstore.MockNetworkChangesStore, *mockstore.
-		MockDeviceChangesStore) {
+const (
+	Device1                   = "Device1"
+	DeviceVersion             = "1.0.0"
+	DeviceType                = "TestDevice"
+	NetworkChange1 network.ID = "NetworkChange1"
+)
 
-	var change1 *change.Change
-	var device1config *store.Configuration
-	var mgrTest *Manager
+func setUp(t *testing.T) (*Manager, *mockstore.MockDeviceStore, *mockstore.MockNetworkChangesStore, *mockstore.MockDeviceChangesStore) {
 
+	log.SetOutput(os.Stdout)
```

Comment:
an alternative to this is t.log()

Reply:
This was in the original tests, I just moved it from a Main() func into this set up one. I think the idea is to redirect all of the logging output of the entire system to the console for these tests.